doc/concepts/data-concepts/repo.md

```diff
@@ -0,0 +1,33 @@
+# Repository
+
+A Pachyderm repository is a location where you store your data inside
+Pachyderm. A Pachyderm repository is a top-level data object that contains
+files and folders. Similar to Git, a Pachyderm repository tracks all
+changes to the data and creates a history of data modifications that you
+can access and review. You can store any type of file in a Pachyderm repo,
+including binary and plain text files.
+
+Unlike a Git repository that stores history in a `.git` file in your copy
+of a Git repo, Pachyderm stores the history of your commits in a centralized
+location in the etcd key-value store. Because of that, you do not run into
+merge conflicts as you often do with Git commits when you try to merge
+your `.git` history with the master copy of the repo. With large datatsets
+resolving a merge conflict might not be possible.
+
+A Pachyderm repository is the first entity that you configure to create
+a standard pipeline. You can create a repository by running the
+`pachctl create repo <name>` command or by using the Pachyderm UI. After
+creating the repository, you can add your data by using the
+`pachctl put file <repo@branch:/path><path-to-filesystem>` command. The
```

Comment:
> `pachctl put file <repo@branch:/path><path-to-filesystem>`

This command isn't quite right. What are you going for here? You can probably simplify this by just sayinf "by using the `put file` command. 

Note: pachctl is just the command line interface. Technically, you can use the python client, the UI, the Go client, or the grpc API directly to add data to pachyderm. I probably missed this in a bunch of other places above, but we don't need to actually have copy/paste-able commands in these case. Just say, "you add data to a repo by using 'put file'" and then they can to go elsewhere (e,g. "getting you data in pachyderm" doc to get the actual command syntax for put file because otherwise this becomes a lot of superfluous information when we're really just talking about repos.